api/src/test/java/org/openmrs/util/databasechange/Database1_9_7UpgradeIT.java

```diff
@@ -64,7 +66,8 @@
 	
 	private DatabaseUpgradeTestUtil upgradeTestUtil;
 	
-	private static File testAppDataDir;
+	@ClassRule
+	public static TemporaryFolder testAppDataDir = TemporaryFolder.builder().assureDeletion().build();
```

Comment:
you need to remove 125-128. please remember that JUnit is now creating and deleting the directory. So we do not need any code that creates/deletes athw testAppDataDir anymore